### ISSUE TITLE
fcn8s model trained on COCO

### ISSUE THREAD

**Issue by swamiviv** [torrvision/crfasrnn#73]
Hi,

Is it possible to release the fcn8s model that is trained on COCO and produces 69.85% on the reduced pascal voc 2012 validation set ? I would like to use that model to train the RNN layer.

**Comment by sadeepj** [comment]
Hi swamiviv,

We've provided the best model described in our paper, which achieves roughly 73% IOU on the reduced validation set (see https://github.com/torrvision/crfasrnn/blob/master/python-scripts/download_trained_model.sh). Unfortunately, it is not possible to provide and maintain all intermediate models described in the paper as it would increase our workload of maintaining this open source project.
